### PR TITLE
Documentation: Update release notes and support for visual/low-code technologies

### PR DESCRIPTION
**Pull request by dennis-sig**

### CODE REVIEW

File: docs/reference/release-notes.md
Comment (MichielCuijpers) [inline docs/reference/release-notes.md:1]: identofy -> identify
filterm -> filter
publush -> publish

Comment (dennis-sig) [reply docs/reference/release-notes.md:1]: Wow :O

Comment (dennis-sig) [reply docs/reference/release-notes.md:1]: Sorry about typo land, should be fixed now.

**Review by olimoli9160 — APPROVED** [review]

File: docs/reference/release-notes.md
Comment (olimoli9160) [reply docs/reference/release-notes.md:1]: I approved, only thing is we're still awaiting the release of the new Churn card. If it is not there on Monday, I would say we leave it for the next release notes.

Comment (dennis-sig) [reply docs/reference/release-notes.md:1]: I took it out of the release notes, you can re-add it once it's available.

**Pull request merged**